Makefile

```diff
@@ -1,7 +1,7 @@
 # A bug(?) in the configparser install means it does not set up backports/
 # the way we need to, according to https://pypi.python.org/pypi/backports.
 # We fix that manually.
-deps vendor_deps: check_setup
+py_deps py_vendor_deps: check_setup
```

Comment:
This looks like it's js deps too!  You could either split it further, or merge everything into one rule.

Reply:
Agreed. I change the name back to deps, and add "go_deps" to be one new dependence of it.